app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -2,29 +2,33 @@
   display: flex;
   flex-wrap: wrap;
   margin-top: $gutter-half;
+
+  @include media(tablet) {
+    margin-right: -25px;
+  }
 }
 
 .app-c-highlight-boxes__item-wrapper {
   @include core-19;
+  @include box-sizing(border-box);
   list-style-type: none;
-  padding: 5px 25px $gutter-two-thirds 0;
-  width: 33%;
+  padding: 5px 10px 25px 0;
+  width: 100%;
   min-height: 150px;
```

Comment:
I'm still not certain a min-height is necessary, due to the way flexbox works. Unless this min height is to try to make them all look a similar height on mobile?

Reply:
Sorry, didn't spot this comment on the last PR. I agree - it isn't really needed (I think it was implemented for mobile, as you say, but I don't think they need to be a similar height on mobile because of the way they're stacked). Will remove 👍